product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

```diff
@@ -96,7 +96,12 @@ The following options aren't available when creating your cluster:
        !!!tip
        To maximize your disk size for AWS, select R5b as your instance and then io2 Block Express as your storage to get a maximum disk size of 64 TB and 256,000 IOPS.
   
-1.  In the **Storage** section, from the **Volume Type** list, select your volume type. 
+1.  In the **Storage** section:
+
+    By default, the **Database Storage** volume includes the storage volume for Write-Ahead (WAL) Logs. If your database is OLTP and have more WAL log generation then you can allocate separate storage volume for the WAL logs. To allocate separate storage volume for WAL logs, select the check-box before **Use a separate storage volume for Write-Ahead Logs**. Select the Volume Type, Size, IOPS, and Disk Throughput separately for **Database Storage** and **Write-Ahead Logs Storage**.
```

Comment:
my suggested edit:

By default, the **Database Storage** volume stores Postgres data and the Write-Ahead Log (WAL) together. If you want to improve write performance for WAL files, you can allocate a separate storage volume for the WAL by selecting the check-box before **Use a separate storage volume for Write-Ahead Logs**. Then select the Volume Type, Size, IOPS, and Disk Throughput separately for **Database Storage** and **Write-Ahead Logs Storage**. If you allocate a separate storage volume for the WAL, you will have to pay cloud infrastructure costs for the second volume.